### ISSUE TITLE
can't add labels to tasks uising talkback screen reader

### ISSUE THREAD

**Issue by rilygreat** [HabitRPG/habitica-android#2187]
> Device: Xiaomi 2409BRN2CA
> Android Version: 35
> AppVersion: Sürüm 4.9.1 (17370)
> User ID: 42ef6728-0aca-4dae-96b6-c6fd598c1cfe
> 
> I am a visually impaired user of your application and I use the TalkBack screen reader.
> ​While I am able to add labels to my tasks on the desktop version, I am having trouble doing the same on Android. My screen reader announces "labels," but I cannot find an option to actually add or select them.
